src/cbapi/__init__.py

```diff
@@ -6,7 +6,7 @@
 __author__ = 'Carbon Black Developer Network'
 __license__ = 'MIT'
 __copyright__ = 'Copyright 2018-2019 VMware Carbon Black'
```

Comment:
This copyright needs updated

Reply:
Wonder why it wasn't updated with Release 1.6.2? Oh well, updated it.